src/lib/utils/os_utils.cpp

```diff
@@ -15,6 +15,10 @@
 #include <chrono>
 #include <cstdlib>
 
+#if !defined(BOTAN_TARGET_OS_HAS_SYSCONF)
+  #include <thread>
```

Comment:
This include should be guarded by `BOTAN_TARGET_OS_HAS_THREADS`  since there are platforms that have neither POSIX APIs nor thread support.

Reply:
Condition here is inverted (should be HAS_THREADS, not !HAS_THREADS)

Reply:
Missing include due to this is causing Windows CI to fail.

Reply:
AH sorry ... typo.